data-raw/run_and_compare_templates.R

```diff
@@ -0,0 +1,67 @@
+library(purrr)
+library(cli)
+library(stringr)
+library(devtools)
+devtools::install_github("pharmaverse/pharmaversesdtm", ref = "main")
```

Comment:
is it possible to just use remotes or something else?  devtools is such a huge package to install

Reply:
sorry you mentioned the runtime of the action and i started looking more closely.